app/src/main/java/com/nextcloud/talk/controllers/bottomsheet/CallMenuController.java

```diff
@@ -123,7 +123,18 @@ private void prepareMenu() {
         menuItems = new ArrayList<>();
 
         if (menuType.equals(MenuType.REGULAR)) {
-            menuItems.add(new MenuItem(getResources().getString(R.string.nc_configure_room), 0, null));
+            if (room.displayName != null && room.displayName.length()>0) {
```

Comment:
TextUtils.isEmpty(room.getDisplayName()) ? :)

Same in all other places.

Reply:
fixed https://github.com/nextcloud/talk-android/pull/204/commits/2f3c2c0f17b39110e3187d8e70e2924eab89189e